### DISCUSSION TITLE
Matching Algorithm2

### DISCUSSION THREAD

**Discussion by protruser** [mannmi/KoreanWonDetection#39] [Ideas]
I separated the bill into three parts like 02, 03 and 06. But it is not work well, and hard to detect it.
Due to the hidden images, it still gives an error like

```
res_06 = cv2.matchTemplate(roi_06, template_06, cv2.TM_CCOEFF_NORMED)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
cv2.error: OpenCV(4.10.0) D:\a\opencv-python\opencv-python\opencv\modules\imgproc\src\templmatch.cpp:1175: error: (-215:Assertion failed) *img.size().height <= *templ.size().height && *img.size().width <= *templ.size().width in function 'cv::matchTemplate'
```

we have to solve this problem of how to separate the bill and get features of hidden images

This is my idea.
![image](https://github.com/user-attachments/assets/68532818-993c-4ded-87d8-43b698aed383)

You can share the comments at any time!!

**Comment by gogog01-29-2021** [comment]
It is really nice try of implementation and I think it should be also be an issue to prepare pull requests
